src/gameplay_v2/roles/components/hideFromSpy.ts

```diff
@@ -0,0 +1,7 @@
+import Component from './component';
+
+//Oberon's component
+export class hideFromSpyC extends Component {
+  name = 'hideFromSpyC';
+  data = {};
+}
```

Comment:
As discussed, let's remove this! Oberon hiding should just be implemented in the seeSpies system